lib/util/src/alloc.c

```diff
@@ -16,8 +16,8 @@ alloc_aligned(size_t size, size_t align)
     if ((align & (align - 1)) || align > UINT32_MAX || size > UINT32_MAX)
         return NULL;
 
-    if (align < _Alignof(max_align_t))
-        align = _Alignof(max_align_t);
+    if (align < alignof(max_align_t))
```

Comment:
When is the double underscore version used versus this plain variant?

Reply:
See my previous note.